srcpkgs/uucp/patches/uucp-org.patch

```diff
@@ -0,0 +1,8833 @@
+-news packages are `C-news' and `INN'.  Both are available in source
+-code form.
+-
+-   Configuring and using mail delivery agents is a notoriously complex
+-topic, and I will not be discussing it here.  Configuring news systems
+-is usually simpler, but I will not be discussing that either.  I will
+-merely describe the interactions between the mail and news systems and
+-UUCP.
++`u'
++     void (function return values only)
+ 
+-   A mail or news system interacts with UUCP in two ways: sending and
+-receiving.
++`z'
++     character string
+ 
+-* Menu:
++   A generic pointer (`p') is sometimes a `void *', sometimes a
++function pointer in which case the prefix is pf, and sometimes a pointer
++to another type, in which case the next character is the type to which
++it points (pf is overloaded).
+ 
+-* Sending mail or news::        Sending mail or news via UUCP
+-* Receiving mail or news::      Receiving mail or news via UUCP
++   An array of strings (`char *[]') would be named `az' (array of
++string).  If this array were passed to a function, the function
++parameter would be named `paz' (pointer to array of string).
+ 
+-
+-File: uucp.info,  Node: Sending mail or news,  Next: Receiving mail or news,  Prev: Mail and News,  Up: Mail and News
++   Note that the variable name prefixes do not necessarily indicate the
++type of the variable.  For example, a variable prefixed with `i' may be
++int, long or short.  Similarly, a variable prefixed with `b' may be a
++char or an int; for example, the return value of `getchar' would be
++caught in an int variable prefixed with `b'.
+ 
+-Sending mail or news via UUCP
+------------------------------
++   For a non-local variable (extern or file static), the first character
++after the type prefix is capitalized.
+ 
+-   When mail is to be sent from your machine to another machine via
+-UUCP, the mail delivery agent will invoke `uux'.  It will generally run
+-a command such as `uux - SYSTEM!rmail ADDRESS', where SYSTEM is the
+-remote system to which the mail is being sent.  It may pass other
+-options to `uux', such as `-r' or `-g' (*note Invoking uux::).
+-
+-   The news system also invokes `uux' in order to transfer articles to
+-another system.  The only difference is that news will use `uux' to
+-invoke `rnews' on the remote system, rather than `rmail'.
+-
+-   You should arrange for your mail and news systems to invoke the
+-Taylor UUCP version of `uux'.  If you only have Taylor UUCP, or if you
+-simply replace any existing version of `uux' with the Taylor UUCP
+-version, this will probably happen automatically.  However, if you have
+-two UUCP packages installed on your system, you will probably have to
+-modify the mail and news configuration files in some way.
+-
+-   Actually, if both the system UUCP and Taylor UUCP are using the same
+-spool directory format, the system `uux' will probably work fine with
+-the Taylor `uucico' (the reverse is not the case: the Taylor `uux'
+-requires th
```

Comment:
8 thousand line patch? Coudn't debian make new release?

Reply:
It seems that the original author did not fix simple errors, Debian only made this big patch, to fix trivial things. I agree should have released a version.